### PR TITLE
Update dependencies

### LINKED ISSUE #700: Update dependencies
We have 11 (and counting) issues open by greenkeep: https://github.com/CraveFood/farmblocks/issues?q=is%3Aissue+is%3Aopen+label%3Agreenkeeper

Let's update everything we can.

### PR DESCRIPTION
**Pull request by fgameiro**
## Closes Issues

* Closes #700

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

**Comment by fgameiro** [comment]
I'm going to update React and Storybook to newer versions too.

**Comment by vNakamura** [comment]
> I'm going to update React and Storybook to newer versions too.

E lá vamos nós

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "jest-enzyme": "^7.0.0",
     "jest-styled-components": "^6.2.2",
     "lerna": "^3.10.5",
-    "prettier": "^1.13.4",
+    "prettier": "^1.16.4",
```
Comment (alcferreira) [inline package.json]: please update it on `pre-commit-config.yaml` as well

File: package.json [hunk 2]
```diff
@@ -86,9 +86,9 @@
     "lerna": "^3.13.1",
     "prettier": "^1.16.4",
     "prop-types": "^15.7.2",
-    "react": "^16.8.2",
-    "react-dom": "^16.8.2",
-    "react-test-renderer": "^16.8.2",
+    "react": "^16.8.4",
```
Comment (alcferreira) [inline package.json]: please change it on `/packages/scaffold/src/templates/package.js` as well

**Comment by alcferreira** [comment]
There are other dependencies that are outdated. Can you please update them as well? Thanks 

image

File: package.json (hunk 1 above)
Comment (fgameiro) [reply package.json]: on `pre-commit-config.yaml` we already have `rev: "1.16.4"` for prettier 🤔

Comment (alcferreira) [reply package.json]: Oh, sorry! I must have opened the webapp file which is 1.13.6

File: packages/alert/package.json [hunk 3]
```diff
@@ -32,7 +27,7 @@
   "homepage": "https://github.com/CraveFood/farmblocks#readme",
   "peerDependencies": {
     "prop-types": "^15.7.2",
-    "react": "^16.8.2",
+    "react": "^16.8.3",
```
Comment (vNakamura) [inline packages/alert/package.json]: The root `package.json` has `"react": "^16.8.4"`
But inside the packages, it's `"react": "^16.8.3"`

**Pull request merged**